src/components/controls/Dropdown.tsx

```diff
@@ -10,7 +10,7 @@ interface IDropdownBaseProps {
     label?: string;
     selected: string;
     disabled?: boolean;
-    onChange(selectedItem: string): void;
+    onChange?: (selectedItem: string) => void;
```

Comment:
Why onChange is optional? Shouldn't dropdown always require onChange?

Reply:
We have permanent disabled dropbox, that we necessary don't want to specify onChange methods for.